patches.xen/Revert-xenbus-track-caller-request-id.patch

```diff
@@ -0,0 +1,60 @@
+From 137e4287ebd6ccdc0982c622825c90bba940f7cf Mon Sep 17 00:00:00 2001
+From: Simon Gaiser <simon@invisiblethingslab.com>
+Date: Tue, 27 Feb 2018 18:14:15 +0100
+Subject: [PATCH] Revert "xenbus: track caller request id"
```

Comment:
So, I see there is still no proper fix for this... Maybe time to ask Juergen again?

Reply:
Yeah, will resend the patches with the changes which are clear and see what they say.

Reply:
Was this for the issue where network devices wouldn't come back after suspending? If so, it looks like it *might* be fixed in the upcoming 4.14.24 version coming out this weekend:

https://lkml.org/lkml/2018/3/2/169